addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2265,11 +2265,13 @@ options.registry.topMenuColor = options.Class.extend({
      */
     async selectStyle(previewMode, widgetValue, params) {
         await this._super(...arguments);
-        const className = widgetValue ? (params.colorPrefix + widgetValue) : '';
+        if (widgetValue && !ColorpickerWidget.isCSSColor(widgetValue)) {
+            widgetValue = params.colorPrefix + widgetValue;
+        }
         await new Promise((resolve, reject) => {
             this.trigger_up('action_demand', {
                 actionName: 'toggle_page_option',
-                params: [{name: 'header_color', value: className}],
+                params: [{name: 'header_color', value: widgetValue}],
```

Comment:
Maybe be more specific in the commit message: (before the color choice was limited by ... etc.)

Reply:
I have a bug if I try to set the opacity of the bg color in the colorpicker (infinite loader)

Reply:
FIXED